src/core/common/permissions/types.ts

```diff
@@ -71,17 +71,17 @@ export const isModerator = (user: Readonly<User>): user is Moderator => {
   return user.role === "MODERATOR";
 };
 
-export const isSiteModerator = (user: Readonly<User>): user is SiteModerator =>
+export const isSiteModerator = (user: User): user is SiteModerator =>
   user.role === "MODERATOR" && isSiteModerationScoped(user.moderationScopes);
 
-export const isOrgModerator = (user: Readonly<User>): user is OrgModerator =>
+export const isOrgModerator = (user: User): user is OrgModerator =>
   user.role === "MODERATOR" && !isSiteModerationScoped(user.moderationScopes);
 
-export const isLTEModerator = (user: Readonly<User>): user is LTEModerator => {
+export const isLTEModerator = (user: User): user is LTEModerator => {
```

Comment:
Just curious, what is an `LTEModerator` does LTE stand for `less than or equal to` or something similar?

Reply:
Yes, exactly, that would be worth a doc string